eng/packages/http-client-csharp-mgmt/docs/emitter.md

```diff
@@ -0,0 +1,134 @@
+---
```

Comment:
is this duplicated with README.md?

Reply:
Yeah, there is no way to avoid it. /cc @timotheeguerin

Reply:
if so, should we have one for http-client-csharp as well?

Reply:
Hmm yeah not sure what happened there.

Reply:
Ok added.